### PR TITLE
Clean up code

### PR DESCRIPTION
**Pull request by felissaf**

### CODE REVIEW

**Review by charliemoweng — COMMENTED** [review]
Looks good! Thanks for the effort!

File: src/main/java/seedu/address/logic/commands/ClearCommand.java [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
 
 /**
- * Clears the address book.
+ * Clears TAB.
  */
 public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
-    public static final String MESSAGE_SUCCESS = "StudentId book has been cleared!";
+    public static final String MESSAGE_SUCCESS = "Teaching Assistant's Buddy has been cleared!";
```
Comment (charliemoweng) [inline src/main/java/seedu/address/logic/commands/ClearCommand.java:14]: There is a merge conflict here. We need to decide whether we want the full name `Teaching Assistant Buddy` or the acronym `TAB`.

Comment (felissaf) [reply src/main/java/seedu/address/logic/commands/ClearCommand.java]: Ah okayy I changed it to TAB hehe

File: src/main/java/seedu/address/model/TeachingAssistantBuddy.java [hunk 2]
```diff
@@ -248,7 +248,7 @@ public void removeStudent(Student key) {
 
     @Override
     public String toString() {
-        return students.asUnmodifiableObservableList().size() + " persons";
+        return students.asUnmodifiableObservableList().size() + " students";
         // TODO: refine later
```
Comment (Ruofan1023) [inline src/main/java/seedu/address/model/TeachingAssistantBuddy.java:252]: What do we need to refine here?

**Review by Ruofan1023 — APPROVED** [review]
Good Job! LGTM

**Review by tanjoewel — APPROVED** [review]
Looks good! Thanks for doing this!

File: src/main/java/seedu/address/model/TeachingAssistantBuddy.java (hunk 2 above)
Comment (felissaf) [reply src/main/java/seedu/address/model/TeachingAssistantBuddy.java:252]: not sure tbh I'll just remove the comment ahhaha thanks!!

**Pull request merged**